templates/master.yaml.tmpl

```diff
@@ -2086,6 +2086,58 @@ storage:
 
               echo "Successfully resolved domain $domain"
             done
+    - path: /opt/install-debug-tools
+      filesystem: root
+      mode: 0544
+      contents:
+        inline: |
+            #!/bin/bash
+            set -eu
+
+            # download etcdctl
+            ETCD_VERSION=v0.4.5
+            wget https://github.com/etcd-io/etcd/releases/download/${ETCD_VERSION}/etcd-${ETCD_VERSION}-linux-amd64.tar.gz
+            tar xvf etcd-${ETCD_VERSION}-linux-amd64.tar.gz
+            mv etcd-${ETCD_VERSION}-linux-amd64/etcdctl /opt/bin/etcdctl
+            chmod +x /opt/bin/etcdctl
+            rm etcd-${ETCD_VERSION}-linux-amd64.tar.gz
+            rm etcd-${ETCD_VERSION}-linux-amd64 -Rf
```

Comment:
etcdctl is part of CoreOS distribution so I am not so sure we really need to download yet another etcdctl

Reply:
true